docs/source/user-guide/sql/ddl.md

```diff
@@ -156,6 +156,8 @@ CREATE [OR REPLACE] TABLE [IF NOT EXISTS] <b><i>table_name</i></b> AS [SELECT |
 ```sql
 CREATE TABLE IF NOT EXISTS valuetable AS VALUES(1,'HELLO'),(12,'DATAFUSION');
 
+CREATE TABLE IF NOT EXISTS valuetable (c1 INT NOT NULL, c2 VARCHAR NOT NULL) AS VALUES(1,'HELLO'),(12,'DATAFUSION');
```

Comment:
perhaps its worth to mention that by default column is nullable? or amend the example above to 
```
CREATE TABLE IF NOT EXISTS valuetable AS VALUES(1,'HELLO'),(12,'DATAFUSION'),(null, 'blablabla');
```

To highlight the default nullability